.github/workflows/e2e.yaml

```diff
@@ -48,4 +48,4 @@ jobs:
         relayer-tag: "v2.0.0"
         chain-binary: "simd"
         # on regular PRs we won't run interchain account or upgrade tests.
-        test-exclusions: "TestInterchainAccountsTestSuite,TestIncentivizedInterchainAccountsTestSuite,TestUpgradeTestSuite"
+        test-exclusions: "TestInterchainAccountsTestSuite,TestIncentivizedInterchainAccountsTestSuite,TestUpgradeTestSuite,TestUpgradeV6TestSuite"
```

Comment:
this additional exclusion isn't required anymore

Reply:
Nice catch! :D